frontend/packages/pipelines-plugin/src/components/repository/list-page/RepositoryHeader.tsx

```diff
@@ -0,0 +1,62 @@
+import { sortable } from '@patternfly/react-table';
+import { TFunction } from 'i18next';
+import { Kebab } from '@console/internal/components/utils';
+
+export const repositoriesTableColumnClasses = [
+  'pf-u-w-16-on-xl pf-u-w-25-on-lg pf-u-w-33-on-xs', // name
+  'pf-u-w-12-on-xl pf-u-w-20-on-lg pf-u-w-30-on-xs', // Event type
+  'pf-u-w-12-on-xl pf-u-w-20-on-lg pf-u-w-30-on-xs', // Last run
+  'pf-u-w-16-on-xl pf-u-w-25-on-lg pf-u-w-33-on-xs', // Task status
+  'pf-m-hidden pf-m-visible-on-xl', // last run status
+  'pf-m-hidden pf-u-w-12-on-xl pf-u-w-20-on-lg pf-u-w-33-on-xs pf-m-visible-on-xl', // Last run time
+  'pf-m-hidden pf-m-visible-on-xl', // Last run duration
+  Kebab.columnClass,
+];
+
+const RepositoryHeader = (t: TFunction) => () => {
```

Comment:
Instead of passing the t: TFunction around, we could simply import 
`import 118n from 'i18next` and use it as `i18n.t('pipelines-plugin~Name`)

```suggestion
const RepositoryHeader = () => () => {
```